features/not_seeing_refund_payments_and_credit_memos_on_admin_order_view.feature

```diff
@@ -0,0 +1,33 @@
+@refunds
+Feature: Not seeing refund payments and credit memos on admin order view
+    In order to be prevented from having incorrectly generated credit memos or refund payments
+    As an Administrator
+    I want not to see them if process fails on any step
```

Comment:
🚀 
```suggestion
   I don't want to see them if process fails on any step
```
or
```suggestion
    I do not want to see them if process fails on any step
```

Reply:
To be honest, I would for the original sentence 🚀

Reply:
I don't want to generate just one of them?

Reply:
Not seeing them would just mean that we can hide them in UI and job's done 🎉

Reply:
I thought a little bit about a goal as well, and I came up with:

```
In order to have consistent documents and payment in a refunded order
As an Administrator
I don't want to have a credit memo or refund payment generated alone
```

Feel free to comment if you see any danger in such a change 😄 🖖